doc/sphinx-guides/source/installation/config.rst

```diff
@@ -372,6 +372,9 @@ Reported Working S3-Compatible Storage
   **Can be used for quick testing, too:** just use the example values above. Uses the public (read: unsecure and
   possibly slow) https://play.minio.io:9000 service.
 
+`Surf Object Store v2019-10-30 <https://www.surf.nl/en>`_
```

Comment:
Is "Surf Object Store" open source? Can I install it in my data center? I couldn't quickly these answers at the link provided here: https://www.surf.nl/en

Here's what I see:

![Screen Shot 2019-10-30 at 11 47 24 AM](https://user-images.githubusercontent.com/21006/67874259-11999280-fb0b-11e9-938b-0292a6a70b9c.png)


In contrast, I get these answers right away from the Minio link: http://minio.io

Here's a screenshot of that page:

![Screen Shot 2019-10-30 at 11 46 23 AM](https://user-images.githubusercontent.com/21006/67874208-fdee2c00-fb0a-11e9-93e7-30e145c0d54f.png)